workbench-core/environments/src/services/environmentTypeService.test.ts

```diff
@@ -97,23 +97,173 @@ describe('environmentTypeService', () => {
+      const actualResponse = await envTypeService.listEnvironmentTypes({
+        pageSize: 1,
+        paginationToken: validPaginationToken
+      });
 
       // CHECK
       expect(actualResponse).toEqual({ data: [envType] });
     });
+
     test('invalidPaginationToken', async () => {
       // BUILD & OPERATE & CHECK
-      await expect(envTypeService.listEnvironmentTypes(1, 'invalidPaginationToken')).rejects.toThrow(
-        'Invalid paginationToken'
-      );
+      await expect(
+        envTypeService.listEnvironmentTypes({ pageSize: 1, paginationToken: 'invalidPaginationToken' })
+      ).rejects.toThrow('Invalid paginationToken');
+    });
+
+    test('filters by name successfully with an eq function ', async () => {
+      // BUILD
+      const queryItemResponse: QueryCommandOutput = {
+        Items: [marshall(envType)],
+        $metadata: {}
+      };
+      ddbMock
+        .on(QueryCommand, {
+          TableName: TABLE_NAME,
+          IndexName: 'getResourceByName',
+          KeyConditionExpression: '#resourceType = :resourceType AND #name = :name',
+          ExpressionAttributeNames: {
+            '#resourceType': 'resourceType',
+            '#name': 'name'
+          },
+          ExpressionAttributeValues: {
+            ':resourceType': {
+              S: 'envType'
+            },
+            ':name': {
+              S: 'Jupyter Notebook'
+            }
+          }
+        })
+        .resolves(queryItemResponse);
+
+      // OPERATE
+      const actualResponse = await envTypeService.listEnvironmentTypes({
+        pageSize: 1,
+        filter: { name: { eq: 'Jupyter Notebook' } }
+      });
+
+      // CHECK
+      expect(actualResponse).toEqual({ data: [envType] });
+    });
+
+    test('filters by status successfully with gte function ', async () => {
+      // BUILD
+      const queryItemResponse: QueryCommandOutput = {
+        Items: [marshall(envType)],
+        $metadata: {}
+      };
+      ddbMock
+        .on(QueryCommand, {
+          TableName: TABLE_NAME,
+          IndexName: 'getResourceByStatus',
+          KeyConditionExpression: '#resourceType = :resourceType AND #status >= :status',
+          ExpressionAttributeNames: {
+            '#resourceType': 'resourceType',
+            '#status': 'status'
+          },
+          ExpressionAttributeValues: {
+            ':resourceType': {
+              S: 'envType'
+            },
+            ':status': {
+              S: 'A'
+            }
+          }
+        })
+        .resolves(queryItemResponse);
+
+      // OPERATE
+      const actualResponse = await envTypeService.listEnvironmentTypes({
+        pageSize: 1,
+        filter: { status: { gte: 'A' } }
+      });
+
+      // CHECK
+      expect(actualResponse).toEqual({ data: [envType] });
+    });
+
+    test('sort by asc status successfully', async () => {
+      // BUILD
+      const queryItemResponse: QueryCommandOutput = {
```

Comment:
Not blocking: Jest provides a really neat way of letting you run setups within a given context so you don't have to copy and paste as much. It makes the tests cleaner and easier to read. [Here's an example of how its done](https://github.com/aws-solutions/solution-spark-on-aws/blob/develop/workbench-core/accounts/src/services/costCenterService.test.ts#L118). Let me know if you want to chat.